src/components/appBar/AppBar.tsx

```diff
@@ -52,7 +52,7 @@ const AppBar = ({ title, showBackButton = false, topPanelHeight }: Props) => {
   const { i18n, t } = useTranslation();
   const { appBarTitle, systemsList } = useAppBar();
   const location = useLocation();
-  const dropdownDisabled = shouldDropdownBeDisabled(location.pathname);
+  const disabled = shouldSystemChangeBeDisabled(location.pathname);
```

Comment:
@Kasmadei renamed the property because it does not do only "disable dropdown" but also "disable removal of value"

How about `isGlobalSystemSwitchDisabled` ?

Reply:
Agree

Reply:
Changed.